clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -577,7 +577,9 @@ final void fail(long now, Throwable throwable) {
             // this RPC. That is why 'tries' is not incremented.
             if ((throwable instanceof UnsupportedVersionException) &&
                      handleUnsupportedVersionException((UnsupportedVersionException) throwable)) {
-                log.trace("{} attempting protocol downgrade.", this);
+                if (log.isDebugEnabled()) {
```

Comment:
There's no need for this conditional, right?

Reply:
A bit unfortunate that this comment was ignored, it caused a checkstyle error with the new checkstyle:

https://github.com/apache/kafka/pull/5058